setup.md

```diff
@@ -5,6 +5,11 @@ nav_title: Flutter Setup
 permalink: /setup/
 ---
 
+(Please remember, Flutter is in _technology preview_. Thanks
+for trying it out, and please don't hesitate to
+[file issues or bugs](https://github.com/flutter/flutter/issues/new)
+for us.)
```

Comment:
good call. I'll add it to our top banner, right after landing this (and I'll remove this note here)